telos-frontend/README.md

```diff
@@ -1,45 +1,56 @@
-# Getting Started with Create React App
-
 This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
 
-## Available Scripts
+# Prerequisites
+- You have already forked the repository and cloned your fork to your local machine. More information [here](https://gist.github.com/Chaser324/ce0505fbed06b947d962)
+- You have the following installed:
+    - [npm](https://www.npmjs.com/get-npm)
+    - [Node.js](https://nodejs.org/en/)
+    - [Prettier](https://prettier.io/docs/en/install.html)
+    - [Lint](https://www.npmjs.com/package/lint)
 
-In the project directory, you can run:
+# Getting Started
 
-### `npm start`
+1. Open the command prompt and set the working directory as the `3Lancers` directory of your cloned repository. (`cd 3Lancers`)
```

Comment:
can you change the cd command to `cd path/to/3Lancers` just to emphasise that they are navigating to the right folder?